### ISSUE TITLE
"SanitizeFieldsMatcher implements a legacy RSpec matcher protocol"

### ISSUE THREAD

**Issue by thbar** [vjt/sanitize-rails#13]
When using sanitize-rails with RSpec 3.1.0, I get this warning:

```
Deprecation Warnings:

--------------------------------------------------------------------------------
Sanitize::Rails::Matchers::SanitizeFieldsMatcher implements a legacy RSpec matcher
protocol. For the current protocol you should expose the failure messages
via the `failure_message` and `failure_message_when_negated` methods.
(Used from /Volumes/Customer/apps/application/spec/models/my_spec.rb:5:in `block (2 levels) in <top (required)>')
--------------------------------------------------------------------------------
```

**Comment by damien** [comment]
@thbar Could you provide a stacktrace for this? I think I know what might be causing this, but it'd be useful to have a test case to reproduce this before anybody takes a stab at the issue.

**Comment by thbar** [comment]
@damien I will see if I can trace which app this came from (since I maintain several apps). If I see the log again, I will push the full stacktrace here. Thanks!

**Comment by damien** [comment]
No problem. I'd hazard a guess that we just need to update things to work with newer versions of RSpec; it's been a while since the matchesrs in this gem were updated.

**Comment by vjt** [comment]
Fixed in 91a460e5, props to @fabn :beers: :smiley:
